lib/linux_admin/service/systemd_service.rb

```diff
@@ -44,12 +37,23 @@ def restart
     end
 
     def reload
-      Common.run!(Common.cmd(:systemctl), :params => ["reload", name])
+      Common.run!(command_path, :params => ["reload", name])
       self
     end
 
     def status
-      Common.run(Common.cmd(:systemctl), :params => ["status", name]).output
+      Common.run(command_path, :params => ["status", name]).output
+    end
+
+    def show
+      output = Common.run!(command_path, :params => ["show", name]).output
+      output.split("\n").map { |line| line.split("=", 2) }.to_h
```

Comment:
I agree with @Fryguy, but I think I'm going to go @bdunne's route as I'll need to add some additional key/value logic to determine if I should cast or parse the value into some other form.